test/unit/general/iotam-autoregistration-test.js

```diff
@@ -301,6 +389,41 @@ describe('IoT Manager autoregistration', function() {
             });
         });
     });
+    describe('When a new service is created in the IoT Agent', function() {
```

Comment:
I assume that for each new added test for /iot/cgroups there is a test for /iot/services that has been copied and adapter to cover the new URL. Is my assumption correct?

Reply:
Yes, after adding the new functionality I had to ensure the previous functionality doesn't impact therefore similar test cases for /iot/cgroups are added.

Reply:
So there is a 1:1 correspondence between the old and the new test, isn't it?

In that case, I'd suggest adding a mark in a comment just above the `describe()` of the old test with a text like this:

```
// This test will be removed if at the end the /iot/services API (now deprecated) is removed
```

Reply:
Fixed at commit e6cee6a